---
layout: archive
title: " "
permalink: /cv/
author_profile: true
redirect_from:
  - /resume
---

{% include base_path %}<br/>
PDF version is also available [here](https://SaradhaVenkatachalapathy.github.io/files/Saradha_CV.pdf)

## EDUCATION AND WORK EXPERIENCE 
* B.Tech in Biotechnology(with Distinction), SASTRA University, India, 2015
* Ph.D in Mechanobiology, National University of Singapore, 2020 (expected)
* Research Assistant, NUS (Sep 2015 -Jul 2016)   

## SKILLS
* Statistics: Linear algebra,Multivariate analysis,Machine Learning, Clustering,Pattern recognition, Diffusion maps and Psuedotime analysis
* Computer Vision: Segmentation, Feature generation and Particle tracking
* Computational Biology: Next Gen-Sequencing analysis of Microarray, RNA-Seq including single cell RNA-Seq and HiC data       
* Experimental Skills: Microscopy, Tissue engineering and mechanical manipulation of cells

## SELECTED RESEARCH PROJECTS

**Automated feature generator for 3D images**
* Built an in-house automatic image processing pipeline for segmentation and feature generation.
* Developed novel parameters that measure morphology, textural and spatial distribution of objects.
* Integrated multi-domain features such as protein expression, RNA seq and image features for deducing functional links. 
* These algorithms reduced the processing time from 1 week to 12 hours.

**Deconvolving cell variability in cancer**
* Only a subset of cells are activated by cancer signals in engineered breast cancer tissue.
* Developed a linear classifier to predict cell shape with an accuracy of 95%.
* Established the existence of activation primed cell shapes using multimodal-multivariate analysis.
* Demonstrated a causal relationship between cell geometry and activation.

**Time series analysis of reprogramming: Trajectory inference**.
* Aligned, analyzed and visualized the transcriptome during mechanically induced de-differentiation. Performed statistical tests and pathway analysis to characterize the temporal changes in the transcription profile and infer the biological relevance. 
* Implemented pseudotemporal ordering of single cell data to identify variable trajectories during the generation of stem cells. 

**DNA structure informs its function**
* Predicted DNA structure from integrating RNA-Seq and ChIP-Seq data and validated the robustness using experiments and HiC data.
* Identified latent immune cells based on image based DNA structures and clustering large single cell RNA-Seq dataset.

**Cell shape modulates cellular response to stimuli**	
* Aligned, analyzed, visualized and interpreted differential gene expression patterns in RNA-Seq and microarray data. Also performed statistical tests and pathway analysis. 
* Demonstrated the cell shape can modulate the transcriptional response to compressive load and inflammation.

**Identification of dead (Apoptotic) cancer cells in a high-content screen**
* Setup preliminary high content screen to characterize cancer cell survival in the presence of various drugs 
* Developed automatic image feature extraction from high content drug screens on cancer cells.
* Deployed multiple machine learning methods for classifying cancer cells as either dead (apoptotic) or live.
Achieved identification accuracy of over 90%.

## AWARDS AND HONOURS
* Dean’s Merit list of the top 2-10% in the University (2014-2015)
* Inspirational Mentorship Award, NUS High School(2017)  
* Best Oral Presentation Award, International Conference on Genomes and AI (2019)

## PAST RESEARCH EXPERIENCE
* Intern, Biophysics laboratory, Raman Research Institute (RRI) (Dec 2013)
* Medical intern, Raju Hospitals: Kanmani Fertility Clinic (June-July 2013)
* Undergraduate Researcher, Chromatin Epigenetics laboratory, SASTRA (2012-2014)

## MENTORSHIP AND TEACHING EXPERIENCE
* Designed and instructed a workshop session-Image Analysis for dummies
* Teaching Assistance for MATLAB Programming- Bootcamp for Mechanobiology 
* Teaching Assistance for Nuclear Mechanics and Genome Regulation
* Supervised and mentored over 5 students in the lab towards their thesis.

## CONFERENCE: TALKS AND POSTERS
<ul>{% for post in site.talks %}
    {% include archive-single-talk-cv.html %}
  {% endfor %}</ul>
  
## PEER REVIEWED PUBLICATIONS
1. Venkatachalapathy S, Jokhun DS, and Shivashankar GV. Multivariate analysis reveals activation-primed fibroblast geometric states in engineered 3D tumor microenvironments. Mol. Biol. Cell 2020;:mbcE19080420. [PMID: 32023167]
2. Damodaran K*, Venkatachalapathy S*, Alisafaei F, Radhakrishnan AV, Sharma Jokhun D, Shenoy VB, and Shivashankar GV. Compressive force induces reversible chromatin condensation and cell geometry dependent transcriptional response. Mol. Biol. Cell 2018;:mbcE18040256. [PMID: 30256731](*equal contribution)
3. Roy B, Venkatachalapathy S, Ratna P, Wang Y, Jokhun DS, Nagarajan M, and Shivashankar GV. Laterally confined growth of cells induces nuclear reprogramming in the absence of exogenous biochemical factors. Proc. Natl. Acad. Sci. U.S.A. 2018;. [PMID: 29735717]
4. Belyaeva A, Venkatachalapathy S, Nagarajan M, Shivashankar GV, and Uhler C. Network analysis identifies chromosome intermingling regions as regulatory hotspots for transcription. Proc. Natl. Acad. Sci. U.S.A. 2017;. [PMID: 29229825]
5. Mitra A, Venkatachalapathy S, Ratna P, Wang Y, Jokhun DS, and Shivashankar GV. Cell geometry dictates TNFα-induced genome response. Proc. Natl. Acad. Sci. U.S.A. 2017;. [PMID: 28461498]
6. Radhakrishnan AV, Jokhun DS, Venkatachalapathy S, and Shivashankar GV. Nuclear Positioning and Its Translational Dynamics Are Regulated by Cell Geometry. Biophys. J. 2017; 112(9):1920-1928. [PMID: 28494962]
